### PR TITLE
lnls-ans-role-network: update network-manager related packages

### PR DESCRIPTION
**Pull request by xresende**
as @edupcoelho requested.

### CODE REVIEW

File: roles/lnls-ans-role-network/README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ This Ansible role configures some network parameters for Sirius Light Source con
 ---
 # Package versions for Debia. Leave empty for the latest
 pkg_network_manager_version: ""
```
Comment (anacso17) [inline roles/lnls-ans-role-network/README.md]: isn't necessary to update this one to `pkg_version_network_manager` as well?

File: roles/lnls-ans-role-network/defaults/main.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 # Package versions for Debia. Leave empty for the latest
 pkg_network_manager_version: ""
```
Comment (anacso17) [inline roles/lnls-ans-role-network/defaults/main.yml]: same here

File: roles/lnls-ans-role-network/README.md (hunk 1 above)
Comment (xresende) [reply roles/lnls-ans-role-network/README.md]: yes, @anacso17, thanks. i just comitted a fix.

@lerwys , there are a number of undefined package version variables in this role's _defaults/main.yml_:

> pkg_version_network_manager_glib: ""
> pkg_version_libnm_qt_devel: ""
> pkg_version_nm_connection_editor: ""
> pkg_version_libsemanage_python: ""
> pkg_version_policycoreutils_python: ""
> 

 shouldn't we define specific versions in _group_vars/all_ ?

File: roles/lnls-ans-role-network/defaults/main.yml (hunk 2 above)
Comment (xresende) [reply roles/lnls-ans-role-network/defaults/main.yml]: fixed.

**Comment by xresende** [comment]
@lerwys , there are a number of undefined package version variables in this role's defaults/main.yml:

    pkg_version_network_manager_glib: ""
    pkg_version_libnm_qt_devel: ""
    pkg_version_nm_connection_editor: ""
    pkg_version_libsemanage_python: ""
    pkg_version_policycoreutils_python: ""

shouldn't we define specific versions in group_vars/all ?

**Pull request merged**